### PR TITLE
User can choose starting point on map

### PR DESCRIPTION
**Pull request by sundqvistjohan**
[PT - User can choose starting point on map](https://www.pivotaltracker.com/story/show/171079738)

Adds EmbedMap component with google map using google-maps-react
Sets initial location coordinates based on geolocation, then by clicking on map
Adds redux and stores coordinates in application state

GIF

### CODE REVIEW

File: src/components/EmbedMap.jsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+const EmbedMap = props => {
+  const { lat, lng } = props.coords
+
+  const onClickHandler = e => {
+    props.changeCoords({ lat: e.latLng.lat(), lng: e.latLng.lng() });
+  };
+
+  const getCurrentLocation = () => {
+    if (window.navigator && window.navigator.geolocation) {
```
Comment (oliverochman) [inline src/components/EmbedMap.jsx:13]: maybe a redundant question, but what happens if the user does not want to share their location?

File: cypress/support/commands.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Cypress.Commands.add('visitWithMockGeolocation', (latitude = 59.334591, longitude = 18.063240) => {
+  const mockGeolocation = (win, latitude, longitude) => {
+    cy.stub(win.navigator.geolocation, 'getCurrentPosition', cb => {
```
Comment (mantanquzzmo) [inline cypress/support/commands.js:3]: nice nice

File: src/state/store/configureStore.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import { createStore } from 'redux';
+import rootReducer from '../reducers/rootReducer'
+
```
Comment (mantanquzzmo) [inline src/state/store/configureStore.js]: could get rid of this file and move content to index.js

File: src/components/EmbedMap.jsx (hunk 1 above)
Comment (sundqvistjohan) [reply src/components/EmbedMap.jsx:13]: state for coords will be initialState (0, 0), map will center on Google Maps default location (San Franscisco).
Could choose our own position to default too (lat 0, long 0 is in the sea so probably don't want that)

Comment (oliverochman) [reply src/components/EmbedMap.jsx:13]: ok cool

**Review by oliverochman — APPROVED** [review]
LGTM!

File: src/state/store/configureStore.js (hunk 3 above)
Comment (sundqvistjohan) [reply src/state/store/configureStore.js]: Agree, updated

**Pull request merged**